src/data.jl

```diff
@@ -337,6 +378,64 @@ function get_all_gates_dictionary(params::Dict{String, Any}, elementary_gates::A
     
 end
 
+function get_all_CR_gates(params::Dict{String, Any}, elementary_gates::Array{String,1})
+    CR_gates_ids = findall(x -> startswith(x, "CR"), elementary_gates)
+
+    CR_complex = Dict{String, Any}()
+
+    if length(CR_gates_ids) >= 1 
+        for i=1:length(CR_gates_ids)
+
+            gate_type = elementary_gates[CR_gates_ids[i]]
+            # CRP_12
```

Comment:
@lockwo What is CRP_12?

Reply:
Controlled rotation pauli is what I was going for (I'm not sure if that comment is actually necessary)